mypyc/genops.py

```diff
@@ -505,6 +505,14 @@ def __init__(self, ir: ClassIR) -> None:
         self._next_label_reg = None  # type: Optional[Value]
         self._next_label_target = None  # type: Optional[AssignmentTarget]
 
+        # These registers hold the error values for the generator object for the case that the
+        # 'throw' function is called.
+        self._exc_type_reg = None  # type: Optional[Value]
+        self._exc_value_reg = None  # type: Optional[Value]
+        self._exc_exc_tb_reg = None  # type: Optional[Value]
```

Comment:
This all feels pretty heavyweight. Since they all come as a group, I'd make it *one* field of type `Optional[Tuple[Value, Value, Value]]`, and I'd probably skip having getters and setters for it in favor of just doing an assert in the probably one place it'll be needed.